config/playground/helm/dev/helm-opensearch-dashboards.yaml

```diff
@@ -185,4 +185,12 @@ extraObjects: []
 # specify the external plugins to install.
 plugins:
   enabled: true
-  installList: ["https://github.com/BionIT/google-analytics-plugin/releases/download/2.18.0/googleAnalytics-2.18.0.zip"]
\ No newline at end of file
+  installList: ["https://github.com/BionIT/google-analytics-plugin/releases/download/2.18.0/googleAnalytics-2.18.0.zip"]
+
+# Olly II features
```

Comment:
Thanks yuye, have you been verify it from your local? Thanks

Reply:
No. Is there a guide for deploying to the playground website? I cannot find the required credentials [here](https://github.com/opensearch-project/dashboards-anywhere/blob/ff0db9737a093ca51411d4eba0a2d9033c12a04d/config/playground/helm/dev/helm-opensearch.yaml#L30-L37)